benefits/eligibility/analytics.py

```diff
@@ -4,37 +4,45 @@
 from benefits.core import analytics as core
 
 
-class StartedEligibilityEvent(core.Event):
+class EligibilityEvent(core.Event):
+    """Base analytics event for eligibility verification."""
+
+    def __init__(self, request, event_type, eligibility_types):
+        super().__init__(request, event_type)
+        self.update_event_properties(eligibility_types=eligibility_types)
```

Comment:
New base class for `benefits.eligibility` events sets the event property.

Note we expect at some point this may be a list, so property name is plural.